fastlane/jetpack_metadata/es-ES/subtitle.txt

```diff
@@ -1 +0,0 @@
-WordPress más rápido y seguro
```

Comment:
Note to self: Check back on this and the other deleted metadata in a day or so. If still not translated, check with the team whether they'd rather fallback to `en-US` (the content of `jetpack_metadata/default`) or use the previously localized copy.

Reply:
See pcdRpT-Ey-p2

Reply:
Related: pdnsEh-pY-p2#comment-905

I think you might need to implement a [similar fallback that I had to implement for WPAndroid](https://github.com/wordpress-mobile/WordPress-Android/pull/16902) after all.